to.etc.domui/src/main/java/to/etc/domui/dom/HtmlTagRenderer.java

```diff
@@ -678,7 +678,7 @@ public void basicNodeRender(final NodeBase b, final IBrowserOutput o, boolean in
 		}
 
 		String s = getStyleFor(b); 								// Get/recalculate style
-		if(s.length() > 0)
+		if(s.length() > 0 || b.shouldRenderStyles())
```

Comment:
Here we should set/reset the renderStyles flag, more or less like this:

```
if(s.length() > 0 || b.isStyleRendered()) {
  o.attr("style", s);
  if(s.length() == 0)
    clearStyleRendered(); // if we rendered the empty style we can remove it next time
  else
    setStyleRendered(); // otherwise there is stuff there
} 
```